source/lib/ze_libapi.cpp

```diff
@@ -197,6 +197,10 @@ zeInitDrivers(
         return result;
     });
 
+    if (result != ZE_RESULT_SUCCESS) {
+        return result;
```

Comment:
I don't see zeInit with this outer return, why does it not need it but here we do?

Reply:
Let me see if we can check within the init instead, that should result in the same behavior as zeInit.

Reply:
Either or results in the same behavior where the result of the context init is verified. This was missed here in my case.

Reply:
nevermind, the reason why this needs to be checked here is because zeInit did not need to continue to allow the L0 apis to work if it failed, zeInitDrivers needs to allow the L0 apis to work so the additional check needs to be made here for the new api until 2.0.

Reply:
By always checking the result for zeInitDrivers here allows for the driver to be labelled "valid" even if zeInit works and zeInitDrivers does not.